src/hooks/tradingView/useTradingView.ts

```diff
@@ -79,6 +91,35 @@ export const useTradingView = ({
   const hasMarkets = marketIds.length > 0;
   const hasPriceScaleInfo = initialPriceScale !== null || hasMarkets;
 
+  const initializeToggle = useCallback(
+    ({
+      toggleRef,
+      tvWidget,
+      isOn,
+      setToggleOn,
+      label,
+      tooltip,
+    }: {
+      toggleRef: React.MutableRefObject<HTMLElement | null>;
+      tvWidget: TvWidget;
+      isOn: boolean;
+      setToggleOn: Dispatch<SetStateAction<boolean>>;
+      label: string;
+      tooltip: string;
+    }) => {
+      if (toggleRef) {
```

Comment:
so i think `toggleRef` is always defined so we dont technically need to check this here 🤔 (i know our current behavior does this too)